data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+
+echo "Gathering bootstrap journals ..."
+mkdir -p "${ARTIFACTS}/bootstrap/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/bootstrap/journals/${service}.log"
+done
+
+echo "Gathering bootstrap containers ..."
+mkdir -p "${ARTIFACTS}/bootstrap/containers"
+sudo crictl ps --all --quiet | while read -r container
+do
+    container_name="$(sudo crictl ps -a --id "${container}" -v | grep -oP "Name: \\K(.*)")"
+    sudo crictl logs "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.log" || true
+    sudo crictl inspect "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.inspect" || true
+done
+mkdir -p "${ARTIFACTS}/bootstrap/pods"
+podman ps --all --quiet | while read -r container
+do
+    sudo podman logs "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.log"
+    sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
+done
+
+# Collect cluster data
+function queue() {
+    local TARGET="${ARTIFACTS}/${1}"
+    shift
+    # shellcheck disable=SC2155
+    local LIVE="$(jobs | wc -l)"
+    while [[ "${LIVE}" -ge 45 ]]; do
+        sleep 1
+        LIVE="$(jobs | wc -l)"
+    done
+    # echo "${@}"
+    if [[ -n "${FILTER}" ]]; then
+        # shellcheck disable=SC2024
+        sudo "${@}" | "${FILTER}" >"${TARGET}" &
+    else
+        # shellcheck disable=SC2024
+        sudo "${@}" >"${TARGET}" &
+    fi
+}
+mkdir -p "${ARTIFACTS}/control-plane" "${ARTIFACTS}/resources"
+
+echo "Gathering cluster resources ..."
```

Comment:
If the BootStrap API is down will any of these commands (below) run/work? These all use `oc` and this could be problematic if the bootstrap api is down. 

Should we consider other ways to get at this information?